### PR TITLE
Add support for using JSON format when logging to console

### LINKED ISSUE #3427: Add support for using JSON format when logging to console
We're migrating all our services to log in json format. It'd be nice if hono could follow suit.
Just like gelf, quarkus supports logging in json by setting the env var `QUARKUS_LOG_CONSOLE_JSON`, if the dependency `quarkus-logging-json` is added.

### PR DESCRIPTION
**Pull request by vhdirk**
Fixes #3427

### CODE REVIEW

**Review by sophokles73 — COMMENTED** [review]
Can you please also amend the admin guides with some documentation regarding the configuration of using either JSON (console) or Gelf based logging? IMHO this would need to go into https://www.eclipse.org/hono/docs/admin-guide/monitoring-tracing-config/#logging

**Comment by sophokles73** [comment]
@vhdirk any news on this?

**Comment by BobClaerhout** [comment]
Dirk is enjoying some time off this week. I'll ask him to have a look at this again next week

**Comment by sophokles73** [comment]
@vhdirk any news on this? we would like to prepare the 2.2.0 release soon ...

**Comment by vhdirk** [comment]
@sophokles73 My apologies for taking so long. I've just added some documentation for this feature.

File: site/documentation/content/admin-guide/monitoring-tracing-config.md [hunk 1]
```diff
@@ -79,15 +79,23 @@ specific default value that differs from the Quarkus default:
...
 
-The Hono components contain support for sending log messages to a centralized log management system like Graylog,
-Logstash or Fluentd. This is done by means of the [*quarkus-logging-gelf* extension](https://quarkus.io/guides/centralized-log-management),
-using TCP or UDP to send logs in the Graylog Extended Log Format (GELF). The logging configuration can be done using
-environment variables with the `QUARKUS_LOG_HANDLER_GELF_` prefix or using corresponding system properties. See the
+By default, all Hono components will log to the system console and can be configured using environment variables
+with the `QUARKUS_LOG_CONSOLE_` prefix or using corresponding system properties. Logging to a file and syslog is
+also available. See the [configuration reference](https://quarkus.io/guides/logging#quarkus-log-logging-log-config_quarkus.log.console-console-logging)
+for details. Log formatting can be configured by either setting `QUARKUS_LOG_CONSOLE_FORMAT`, or, to
+enable logging in JSON format, by setting `QUARKUS_LOG_CONSOLE_JSON` to `"true"`. Follow the [Quarkus documentation](https://quarkus.io/guides/logging#quarkus-logging-json_quarkus.log.console-json-console-logging)
```
Comment (sophokles73) [inline site/documentation/content/admin-guide/monitoring-tracing-config.md:89]: My understanding of https://quarkus.io/guides/logging#configuration is, that JSON logging will be enabled by default. 

FMPOV it should be disabled by default, though, in order to not break backwards compatibility.

**Review by sophokles73 — CHANGES_REQUESTED** [review]

File: site/documentation/content/admin-guide/monitoring-tracing-config.md (hunk 1 above)
Comment (vhdirk) [reply site/documentation/content/admin-guide/monitoring-tracing-config.md:89]: You're absolutely right. I suppose I could disable it by default by setting the appropriate value here: service-base/src/main/resources/application.properties ?

Comment (vhdirk) [reply site/documentation/content/admin-guide/monitoring-tracing-config.md:89]: done!

File: bom/pom.xml [hunk 2]
```diff
@@ -95,6 +95,7 @@ quarkus.smallrye-health.readiness-path=${health.check.readiness-path}
 quarkus.smallrye-health.startup-path=${health.check.startup-path}
 quarkus.vertx.prefer-native-transport=true
 quarkus.vertx.resolver.cache-max-time-to-live=0
+quarkus.log.console.json=false
```
Comment (sophokles73) [inline bom/pom.xml:98]: can you please move this line up so that it follows the lexical ordering?

Comment (vhdirk) [reply bom/pom.xml]: done :)

**Review by sophokles73 — APPROVED** [review]
LGTM

**Comment by sophokles73** [comment]
Thanks for contributing :+1:

**Pull request merged**